{"name": "test_unsuccessful_login[chrome-fttt-ft]", "status": "passed", "attachments": [{"name": "log", "source": "c7411f46-7694-41ec-90aa-8d53e34bc74b-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_browser", "value": "'chrome'"}, {"name": "username", "value": "'fttt'"}, {"name": "password", "value": "'ft'"}], "start": 1622322039973, "stop": 1622322040506, "uuid": "177e63d2-4f32-4c1f-b8e0-6ddc3bde6017", "historyId": "bce06524281788a71e00192f5e087f3f", "testCaseId": "1ce5752f9bf5ac0dd16171f6da2bde9d", "fullName": "test_cases.test_user_sign_in.Test_UserSignIn#test_unsuccessful_login", "labels": [{"name": "story", "value": "Cloud Login"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "@pytest.mark.usefixtures('add_logs_on_failure', 'get_browser')"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_user_sign_in"}, {"name": "subSuite", "value": "Test_UserSignIn"}, {"name": "host", "value": "DESKTOP-57RCRK3"}, {"name": "thread", "value": "3784-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_user_sign_in"}]}